portfolio/src/main/webapp/index.html

```diff
@@ -4,39 +4,41 @@
     <meta charset="UTF-8">
     <title>Lindsey's Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Chewy">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="introduction">
       <h1>Hey, I'm Lindsey.</h1>
       <h2>It's nice to meet you.</h2>
-      <p>This is me.</p>
+      <h2>This is me.</h2>
       <img src="images/profilepic.JPG"/>
     </div>
-
-    <div id="projects">
+    <p></p>
+    <div id="content">
     <p>Here are some of the projects I've been working on:</p>
     <ul>
       <li>CU@Home - an augmented reality snapchat filter that lets you
-      experience school from wherever you are
+      experience Columbia University's campus from wherever you are
       <li>Fractals with Python Turtle - An interactive program that lets
-      you create different fractals of differeing orders
-      <li>Java SpellChecker - checks to see if your word is an a given
+      you create different fractals of differing orders
+      <li>Java SpellChecker - checks to see if your word is in a given
       dictionary and gives you suggestions if it's not
     </ul>
     </div>
-
-    <div id="links">
+    <p></p>
+    <div id="content">
       <p>Find me on <a href="https://github.com/LindseyWeiskopf">GitHub</a> and <a href="https://www.linkedin.com/in/lindsey-weiskopf">LinkedIn</a>.</p>
     </div>
-    
-    <div id="random-message">
+    <p></p>
```

Comment:
Are you inserting paragraph tags to create spacing? You can group all of this into a single paragraph tag that will likely have the same effect. And if that's not enough space, try adding a class and increasing the line space via css.

Reply:
I fixed this by adding a top and bottom margin for all of the div tags  and got rid of the empty paragraph tags.